ament_cmake_core/cmake/core/package_xml_2_cmake.py

```diff
@@ -111,7 +111,9 @@ def generate_cmake_code(package):
     lines.append('set(_AMENT_PACKAGE_NAME "%s")' % package.name)
     for (k, v) in variables:
         lines.append('set(%s_%s %s)' % (package.name, k, v))
-    return lines
+    # Ensure backslashes are replaced with forward slashes because CMake cannot
+    # parse files with backslashes in it.
+    return [l.replace('\\', '/') for l in lines]
```

Comment:
The problem is that it goes through double evaluation, because this goes into a `.cmake` file which puts it into another `.cmake` file or a Python file or something, which results in one of the CMake files having trouble. It was just easier to convert them all to `/` since that is recommended way to do paths in CMake on Windows.